Path2Communicates/Vizualizations/UI_of_path.py

```diff
@@ -0,0 +1,101 @@
+import tkinter as tk
+import const
+from robot_message import RobotMessage
+
+
+class path_UI:
```

Comment:
```suggestion
class PathVisualization
```
I to samo dla nazwy pliku